index.html

```diff
@@ -0,0 +1,147 @@
+        </ul>
+          </div>
+        
+      </nav>
+    </div>
+  </header>
+  <div id="nav-wrap">
+    <div id="close-button"><img src="img/x-icon.svg" alt="close-button"></div>
+    <div id="nav-m-wrap">
+      <li><a href="index.html">Home</a></li>
+      <li><a href="about-page.html">About</a></li>
+      <li><a href="#courses">Courses</a></li>
+      <li><a href="#instructor">Instrictors</a></li>
+    </div>
+  </div>
+  <main>
+    <section id="hero">
+      <div class="container">
+        <div>
+          
+            <h2 class="fs-secondary-heading fw-bold text-accent-400 ">"Hello! future Devs"</h2>
+            <div class="hero-headline">  
+            <h1 class="hero-headline text-accent-400 ">Break into tech with <br>the tech academy </h1>
+          </div>
+          
+          <div class="offer-text">
+            <p class="fs-600">We offer open enrollment Courses, which means students can begin <br>studying at any time, from anywhere.</p>
+          </div>
+            <h2 class="fs-secondary-heading text-neutral-100 fw-bold ">September- December 2022</h2>
+            <h3 class="fs-600 text-neutral-900">20% off the course fee from September- December 2022!</h2>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section id="courses">
+      <div class="container">
+        <div>
+          <h2 class="title fs-700 text-accent-100 fw-semi-bold" >Our Courses</h2>
+          <div class="guide_bar"></div>
+        </div>
+        <div class="even-columns-course">
+          <div class="course-card">
+            <img class="course-icon" src="img/html-5.png" alt="Html-icon">
+            <h3 class="fw-bold fs-500">HTML</h3>
+            <p class="course-desc">Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+          </div>
+          <div class="course-card">
+            <img class="course-icon"src="img/css-3.png" alt="Css-icon">
+            <h3 class="fw-bold fs-500">CSS</h3>
+            <p class="course-desc">Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+          </div>
+          <div class="course-card">
+            <img class="course-icon" src="img/bootstrap.png" alt="bootstrap-icon">
+            <h3 class="fw-bold fs-500">Bootstrap</h3>
+            <p class="course-desc">Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+          </div>
+          <div class="course-card">
+            <img class="course-icon" class="course-icon" src="img/js.png" alt="js-icon">
+            <h3 class="fw-bold fs-500">JavaScript</h3>
+            <p class="course-desc">Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+          </div>
+          <div class="course-card">
+            <img  class="course-icon" src="img/java.png" alt="java-icon">
+            <h3 class="fw-bold fs-500">Java</h3>
+            <p class="course-desc">Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
![work-section (copy)](https://user-images.githubusercontent.com/93039359/196923184-73ae8b29-082a-4cc0-9509-f42db9596956.png)

- [ ] Kindly check the comments in the image for explanation :+1: